src/redux/saga/getOffice.js

```diff
@@ -0,0 +1,13 @@
+import {takeEvery,call} from 'redux-saga/effects';
+import {GET_BRANDS} from '../constant/type';
+
+function* OfficeList() {
+    yield call(
+        
+    )
```

Comment:
empty function??

Reply:
mam i am going to use this now for get api calling